pkg/clusterbuilder/providers/capa/values/private-cluster_values.yaml

```diff
@@ -57,3 +57,18 @@ global:
         enabled: true
         maxPrice: 0.2960
 
+  apps:
+    certManager:
+      values:
+        serviceAccount:
+          annotations:
+            eks.amazonaws.com/role-arn: "{{ .ClusterName }}-CertManager-Role"
+        giantSwarmClusterIssuer:
+          acme:
+            http01:
+              enabled: false
+            dns01:
+              route53:
+                enabled: true
+                # TODO Use a variable as soon as it is available.
+                region: "eu-north-1"
```

Comment:
Does this do anything? Route53 is global.

The TODO comment isn't actionable for people without context.

Reply:
I just moved the config from default apps config to here https://github.com/giantswarm/cluster-standup-teardown/pull/20#discussion_r1598135566

Reply:
No idea why this is here TBH